resources/lang/en/users.php

```diff
@@ -53,6 +53,7 @@
         '404' => 'User not found! ;_;',
         'age' => ':age years old',
         'current_location' => 'Currently in :location',
+        'dateFormat' => 'MMMM YYYY',
```

Comment:
It's specific to the display of that single field though, which is only used in the user profile (in both client and web)